packages/astro/src/@types/astro.ts

```diff
@@ -1094,6 +1094,29 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		inlineStylesheets?: 'always' | 'auto' | 'never';
+		/**
+		 * @docs
+		 * @name build.concurrency
+		 * @type {boolean}
+		 * @default `1`
+		 * @description
+		 * The number of pages to build in parallel. In most cases, you should stick with the default value of `1`,
+		 * and batch or cache long running tasks like fetch calls or data access to improve the overall rendering time.
+		 *
+		 * Use this option only if the refactors are not possible. If the number is set too high, the page rendering
+		 * may slow down due to insufficient memory resources and because JS is single-threaded.
+		 *
+		 * In the future, Astro may reuse this option to render pages concurrently with multiple threads.
+		 *
+		 * ```js
+		 * {
+		 *   build: {
+		 *     concurrency: 1
```

Comment:
```suggestion
		 *     concurrency: 2
```
We would usually not show the default value here, because no one would configure `1`.